asset/scss/_variables.scss

```diff
@@ -4,3 +4,13 @@
 // ----------------------------
 $themeroot: '/themes/mitlibraries-theme-omeka';
 $network: $themeroot + '/asset/img/mitlib-style/vi-shape7-tp.svg';
+
+// ----------------------------
+// #GRID
+// ----------------------------
+
+// grid - breakpoints
+$bp-screen-sm: 480px;
+$bp-screen-md: 768px;
+$bp-screen-lg: 1024px;
+$bp-screen-xl: 1280px;
```

Comment:
It looks like `$bp-screen-sm` and `$bp-screen-md` are both used exactly once and `bp-screen-lg` and `$bp-screen-xl` are not used at all. Combined with there being a breakpoint we don't define here at all, I'm nos sure if this grid breakpoint definition is helpful or misleading.

Suggestions:
- commend out the breakpoints you suspect we'll use in the future but currently do not use
- add an additional named breakpoint here for our 600px width that we currently use

Reply:
I'll either make an attempt at moving the 600px rules to either small or medium, or I'll define a new intermediate size. The large and xl sizes will be commented out.

This section was a copy-pasta from the style library, but I see logic in not leaving it as-is.